### PR TITLE
solution added

### PR DESCRIPTION
**Pull request by Vosker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,6 +20,24 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.21</version>
+        </dependency>
+
         <dependency>
             <groupId>junit</groupId>
```
Comment (SetMyDream) [inline pom.xml]: think junit redundant here

File: src/main/resources/hibernate.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+    <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
+    <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+    <property name="connection.username">root</property>
+    <property name="connection.password">1234</property>
+    <property name="show_sql">true</property>
+    <property name="hbm2ddl.auto">create-drop</property>
+        <mapping class="com.dev.cinema.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (SetMyDream) [inline src/main/resources/hibernate.cfg.xml]: add empty line here)

**Review by SetMyDream — APPROVED** [review]
looks pretty)

File: pom.xml (hunk 1 above)
Comment (Vosker) [reply pom.xml]: пусть будет пока)

Comment (KseniiaMakarova) [reply pom.xml]: let's remove the dependencies we don't use

and also the test class

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Movie entity", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]: let's create our custom DataProcessingException

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<hibernate-configuration>
+    <session-factory>
+    <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+    <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
+    <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+    <property name="connection.username">root</property>
+    <property name="connection.password">1234</property>
+    <property name="show_sql">true</property>
+    <property name="hbm2ddl.auto">create-drop</property>
```
Comment (KseniiaMakarova) [inline src/main/resources/hibernate.cfg.xml]:
```suggestion
        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
        <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
        <property name="connection.username">root</property>
        <property name="connection.password">1234</property>
        <property name="show_sql">true</property>
        <property name="hbm2ddl.auto">create-drop</property>
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/util/HibernateUtil.java [hunk 5]
```diff
@@ -13,7 +14,7 @@ private static SessionFactory initSessionFactory() {
         try {
             return new Configuration().configure().buildSessionFactory();
         } catch (Exception e) {
-            throw new RuntimeException("Error creating Session Factory", e);
+            throw new DataProcessingException("Error creating Session Factory", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/util/HibernateUtil.java]:
```suggestion
            throw new RuntimeException("Error creating Session Factory", e);
```
this exception has a different nature

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**